VideoLocker/src/main/java/org/edx/mobile/user/DiscussionUser.java

```diff
@@ -0,0 +1,35 @@
+
+package org.edx.mobile.user;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class DiscussionUser implements Serializable {
+
+    @SerializedName("profile")
+    private Profile profile;
```

Comment:
This should probably be declared as `@NonNull`, along with it's getter and setter.